File: 백준/Bronze/1978. 소수 찾기/소수 찾기.c
#define _CRT_SECURE_NO_WARNINGS

#include <stdio.h>
#include <stdlib.h>

int main() {

    int N;
    scanf("%d", &N);

    int num;
    int prime_number = 0;

    for (int i = 0; i < N; i++){
        scanf("%d", &num);

        if (num == 1){
            continue;
        }
        int cnt = 0;
        for (int j = 2; j < num; j++){
            if (num % j == 0){
                cnt++;
            }
        }
        if (cnt == 0){
            prime_number++;
        }
    }

    printf("%d", prime_number);

    return 0;
}

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
n = int(input())
num = list(map(int, input().split(' ')))
prime_number = 0

#소수 구하기 에라토스테네스의 체
for i in num:
    cnt = 0
    if i == 1:
        continue
    for j in range(2, i+1):
            if i % j == 0:
                cnt += 1
    if cnt == 1:
        prime_number += 1

print(prime_number)